src/AirshipBindings.Android.Core/Properties/AssemblyInfo.cs

```diff
@@ -7,4 +7,7 @@
 using Android.App;
 
 // Information about this assembly is defined by the following attributes.
-// Change them to the values specific to your project.
\ No newline at end of file
+// Change them to the values specific to your project.
+
+[assembly: AssemblyTitle("AirshipBindings.Android.Core")]
+[assembly: AssemblyProduct("AirshipBindings.Android.Core")]
```

Comment:
Oops. This is cruft from the various things I was trying to get the bindings to work. It seems like a reasonable thing to add to the bindings, but I haven't done it for the others.

Reply:
If it's cruft should we remove it?

Reply:
Sure.